index.html

```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (2).png" alt=""></a>
+    
+        <h3  class="tonic" >Multi-Post
+            Stories</h3>
+    
+    <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (3).png" alt=""></a>
+    
+        <h3  class="tonic" >Tonic</h3>
+    
+    <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (4).png" alt=""></a>
+    
+        <h3  class="tonic" >Multi-Post
+            Stories </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack2">
+        <h3  class="tonic" >About Myself </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> Hello I’m a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! If you like what you see 
+        and have a project you need coded, don’t hestiate to contact me. </p>
+</div>
+<div class = "social-section">
+    <h4 class="connectmywork"> LET’S CONNECT</p> 
```

Comment:
- [ ] This confuses me. It is opening the tag with h4 and classes it with p tag 😒.